src/other_features/replayDebugger.adoc

```diff
@@ -10,34 +10,26 @@ url: doc/other_features/replayDebugger.html
 == [[ReplayDebuggerOverview]] Overview
 Replay Debugging is a powerful tool that allows you to record the execution of your application and then replay it later. This can be useful for debugging purposes, as it allows you to see exactly what happened during the execution of your application.
 
-Tracing information is already available in 4diac FORTE. However, the current implementation traces a lot of information. Basically, it traces all events, data and also internal data from the function block.
+Tracing information is already available in the 4diac FORTE runtime. However, the current implementation traces a lot of information. Basically, it traces all events, data and also internal data from the function block.
```

Comment:
runtime is not needed here. Also the. Please change all `the 4diac FORTE runtime` back to `4diac FORTE`.